site2/docs/concepts-messaging.md

```diff
@@ -254,9 +254,11 @@ In the diagram below, only **Consumer A-0** is allowed to consume messages.
 
 ### Failover
 
-In *failover* mode, multiple consumers can attach to the same subscription. In failover mode, the broker selects the master consumer based on the priority level and the lexicographical sorting of a consumer name. If two consumers have an identical priority level, the broker selects the master consumer based on the lexicographical sorting. If these two consumers have different priority levels, the broker selects the consumer with a higher priority level as the master consumer. The master consumer is initially the only one receiving messages. When the master consumer disconnects, all (non-acknowledged and subsequent) messages are delivered to the next consumer in line.
+In *failover* mode, multiple consumers can attach to the same subscription. A master consumer will be picked for non-partitioned topic or each partition of partitioned topic. The master consumer is the only one receiving messages. When the master consumer disconnects, all (non-acknowledged and subsequent) messages are delivered to the next consumer in line. 
```

Comment:
In *failover* mode, multiple consumers can attach to the same subscription. A master consumer is picked for topics and receives messages. When the master consumer disconnects, all (non-acknowledged and subsequent) messages are delivered to the next consumer in line.

Reply:
I'll keep the non-partitioned topic or each partition of partitioned topic part cuz I think reader will need this detail here.

Reply:
updated